lib/OpenIDConnect/Repositories/ClientRepository.php

```diff
@@ -22,7 +22,9 @@ public function getClientEntity($clientIdentifier)
         $client = new ClientEntity();
         $client->setIdentifier($tool->oidc_client_id);
         $client->setName($tool->name);
-        $client->setRedirectUri($tool->oidc_redirect_url);
+        $client->setRedirectUri(
```

Comment:
I would suggest **not** to do the `explode` directly in the `setRedirectUri`, but to do it beforehand with empty array handler in order to make sure that the passing argument to the method is always an array!


```suggestion
        $redirect_urls_arr = $tool->oidc_redirect_url ? explode("\n", $tool->oidc_redirect_url) : [];
        $client->setRedirectUri($redirect_urls_arr);
```